oceancypress/student_information_system
# Deep prereq which leads to much better test data, some bug fixes

* mostly test data creation
* moved some views around in schooladmin
* fixed #346 

Testing of test data creation:
* see that Students are now making progress on their required courses
* see that when sections are filled, new sections are opened (look for Sec #2 of something, check capacity of #1)
